x-pack/plugin/searchable-snapshots/qa/rest/src/test/resources/rest-api-spec/test/stats.yml

```diff
@@ -167,11 +167,15 @@ teardown:
   - gte:     { indices.docs.shards.0.0.files.0.cached_bytes_written.sum: 0 }
   - gte:     { indices.docs.shards.0.0.files.0.cached_bytes_written.min: 0 }
   - gte:     { indices.docs.shards.0.0.files.0.cached_bytes_written.max: 0 }
+  - gte:     { indices.docs.shards.0.0.files.0.cached_bytes_written.time_in_nanos: 0 }
```

Comment:
I'm wondering if we should check that it's greater than `0`?

Reply:
No, we can't assert that since `System::nanoTime` might not be accurate enough to advance while we're watching it -- see e.g. https://github.com/elastic/elasticsearch/issues/27371.